4.5.1/Dockerfile

```diff
@@ -0,0 +1,137 @@
+FROM ubuntu:trusty
+
+MAINTAINER Yinlin Chen "ylchen@vt.edu"
+
+
+# Install essential packages
+RUN apt-get update && apt-get install -y \
+	build-essential \
+	curl \
+	maven \
+	openssh-server \
+	software-properties-common \
+	vim \
+	wget \
+	htop tree zsh fish
+
+
+# Install Java 8
+# Define commonly used JAVA_HOME variable
+ENV JAVA_HOME /usr/lib/jvm/java-8-oracle
+
+RUN echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | debconf-set-selections \
+	&& add-apt-repository -y ppa:webupd8team/java \
+	&& apt-get update \
+	&& apt-get install -y oracle-java8-installer \
+	&& rm -rf /var/lib/apt/lists/* \
+	&& rm -rf /var/cache/oracle-jdk8-installer
+
+
+# Install Tomcat 7.
+ENV CATALINA_HOME /usr/local/tomcat7
+ENV PATH $CATALINA_HOME/bin:$PATH
+ENV TOMCAT_MAJOR 7
+ENV TOMCAT_VERSION 7.0.67
+ENV TOMCAT_TGZ_URL http://archive.apache.org/dist/tomcat/tomcat-$TOMCAT_MAJOR/v$TOMCAT_VERSION/bin/apache-tomcat-$TOMCAT_VERSION.tar.gz
+
+RUN mkdir -p "$CATALINA_HOME" \
+	&& cd $CATALINA_HOME \
+	&& set -x \
+	&& curl -fSL "$TOMCAT_TGZ_URL" -o tomcat.tar.gz \
+	&& tar -xvf tomcat.tar.gz --strip-components=1 \
+	&& rm bin/*.bat \
+	&& rm tomcat.tar.gz* \
+	&& useradd -ms /bin/bash tomcat7 \
+	&& sed -i '$i<role rolename="fedoraUser"/>$i<role rolename="fedoraAdmin"/>$i<role rolename="manager-gui"/>$i<user username="testuser" password="password1" roles="fedoraUser"/>$i<user username="adminuser" password="password2" roles="fedoraUser"/>$i<user username="fedoraAdmin" password="secret3" roles="fedoraAdmin"/>$i<user username="fedora4" password="fedora4" roles="manager-gui"/>' /usr/local/tomcat7/conf/tomcat-users.xml
+
+
+# Make the ingest directory
+RUN mkdir /mnt/ingest \
+	&& chown -R tomcat7:tomcat7 /mnt/ingest
+
+VOLUME /mnt/ingest
+
+
+# Install Fedora4
+ENV FEDORA_VERSION 4.5.1
+ENV FEDORA_TAG 4.5.1
+
+COPY fcrepo.war /root/
+
+RUN mkdir -p /usr/local/tomcat7/fcrepo4-data \
+	&& chown tomcat7:tomcat7 /usr/local/tomcat7/fcrepo4-data \
+	&& chmod g-w /usr/local/tomcat7/fcrepo4-data \
+	&& cd /tmp \
+	&& cp /root/fcrepo.war /usr/local/tomcat7/webapps/fcrepo.war \
```

Comment:
please change to 
&& curl -fSL https://github.com/fcrepo4-exts/fcrepo-webapp-plus/releases/download/fcrepo-webapp-plus-$FEDORA_TAG/fcrepo-webapp-plus-$FEDORA_VERSION.war -o fcrepo.war \
&& cp fcrepo.war /usr/local/tomcat7/webapps/fcrepo.war \